tests/rptest/clients/rpk.py

```diff
@@ -318,7 +318,10 @@ def describe_topic_configs(self, topic):
 
     def alter_topic_config(self, topic, set_key, set_value):
         cmd = ['alter-config', topic, "--set", f"{set_key}={set_value}"]
-        self._run_topic(cmd)
+        out = self._run_topic(cmd)
```

Comment:
Looks like `replication_factor_change_test` needs to swallow this exception now

Reply:
Fixed.